aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -335,6 +335,38 @@ class QConv2dInt8 final : public c10::OperatorKernel {
     // Force output format to be NHWC
     // TODO: consider preserving input format
     // TODO: add MemoryFormat::ChannelsLast here once perf is fixed
+    auto input_scale = input_contig.q_scale();
+
+    // Re-quantizing the bias based on input scale and weight scale.
+    if (!pack_ptr.input_scale.has_value()) {
```

Comment:
you not just check the value presence, you need to compare the actual value here